app/src/main/java/com/gmail/mxwild/newhabit/moviedetail/FragmentMovieDetails.kt

```diff
@@ -31,16 +35,28 @@ class FragmentMoviesDetails : Fragment() {
             parentFragmentManager.popBackStack()
         }
 
+        movie = arguments?.getParcelable(MOVIE_OBJECT)
+        val viewModelFactory = MovieDetailsViewModelFactory(movie)
```

Comment:
Why to have view model if you already have data here?

Reply:
Fix it